modules/openid_auth/app/controllers/openid_auth/v1/validation_controller.rb

```diff
@@ -6,7 +6,7 @@
 module OpenidAuth
   module V1
     class ValidationController < ApplicationController
-      before_action :validate_user
+      before_action :validate_user, :validate_launch_context
```

Comment:
How would a patient token pass this check?

Reply:
token type check added